client/src/components/FoodSeeker/SearchResults/AdvancedFilters/FilterButtons/PantryButton.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import AdvancedFilterButton from '../AdvancedFilterButton';
+const PantryButton = ({
+    toggleCategory,
+    isSelected,
+}) => {
+    const PantryLocator = () => (
+        <svg width="12" height="16" viewBox="0 0 12 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+            <path d="M0 5.77444C0 8.96357 6 16 6 16C6 16 12 8.96357 12 5.77444C12 2.5853 9.31371 0 6 0C2.68629 0 0 2.5853 0 5.77444Z" fill="#0A7572" />
+            <path d="M6 9.6904C5.78136 9.8885 5.3546 9.99978 4.98787 9.99978C3.72708 9.99978 2 8.1769 2 6.55293C2 4.92896 2.82051 3.77743 4.0813 3.77743C4.76523 3.77743 5.31707 3.84509 5.61176 4.13002L5.60625 4.11208C5.60625 4.11208 5.37861 3.31844 5.00638 3.15173C5.21201 3.06205 5.59275 2.81055 5.59275 2.81055C5.59275 2.81055 6.17911 3.4607 6.17911 4.1129V4.13002C6.61238 3.819 7.13021 3.62661 7.9182 3.62661C9.17899 3.62661 10 4.92896 10 6.55293C10 8.1769 8.21138 9.99978 6.95059 9.99978C6.58086 9.99978 6.21964 9.89217 6 9.6904Z" fill="white" />
+            <path d="M7.57233 3.04881C7.23612 3.50087 6.49716 3.6366 6.49716 3.6366C6.49716 3.6366 6.30304 3.03984 6.63925 2.5882C6.97546 2.13655 7.71392 2 7.71392 2C7.71392 2 7.90804 2.59676 7.57233 3.04881Z" fill="white" />
+        </svg>
+    );
```

Comment:
should we move the icons to the `icons` folder? so we can reuse them 🤔

Reply:
Moved the two locator icons to the /icons folder.